src/prop/cnf_stream.h

```diff
@@ -211,6 +211,8 @@ class CnfStream {
                                 ProofRule proof_id,
                                 TNode from = TNode::null()) = 0;
 
+  inline SatSolver* getSatSolver() { return d_satSolver; };
```

Comment:
I doesn't look like this is being used. Am I missing something?

Reply:
Unresolved, since still not removed.